yesod-bin/AddHandler.hs

```diff
@@ -78,7 +79,11 @@ addHandlerFiles cabal (name, handlerFile) pattern methods = do
     modify cabal $ fixCabal name
```

Comment:
I believe `fixCabal` also needs to be modified so that it adds the Spec module to the test suite.